iphone/CoreApi/CoreApi.xcodeproj/project.pbxproj

```diff
@@ -997,6 +993,7 @@
 				CLANG_WARN_OBJC_IMPLICIT_RETAIN_SELF = YES;
 				CLANG_WARN_OBJC_LITERAL_CONVERSION = YES;
 				CLANG_WARN_OBJC_ROOT_CLASS = YES_ERROR;
+				CLANG_WARN_QUOTED_INCLUDE_IN_FRAMEWORK_HEADER = NO;
```

Comment:
BTW, why don't we put all this CLANG_WARN_XXX into common-debug/common-release ?
Here and in every other pbxproj ..

Reply:
I'm a newbie in Xcode project. Nice to learn that there is a common file which can override settings for all project. I will try to move all common settings to  common-debug/common-release. Thanks for the advice!